apps/token/src/routes/governance/propose/update-market/propose-update-market.spec.tsx

```diff
@@ -0,0 +1,182 @@
+import { MockedProvider } from '@apollo/client/testing';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { VegaWalletContext } from '@vegaprotocol/wallet';
+import { AppStateProvider } from '../../../../contexts/app-state/app-state-provider';
+import {
+  mockWalletContext,
+  networkParamsQueryMock,
+} from '../../test-helpers/mocks';
+import { ProposeUpdateMarket } from './propose-update-market';
+
+jest.mock('../../../../hooks/use-network-param.tsx', () => ({
+  useNetworkParamWithKeys: () => ({
+    data: [
+      {
+        __typename: 'NetworkParameter',
+        key: 'governance.proposal.updateMarket.maxClose',
+        value: '8760h0m0s',
+      },
+      {
+        __typename: 'NetworkParameter',
+        key: 'governance.proposal.updateMarket.maxEnact',
+        value: '8760h0m0s',
+      },
+      {
+        __typename: 'NetworkParameter',
+        key: 'governance.proposal.updateMarket.minClose',
+        value: '1h0m0s',
+      },
+      {
+        __typename: 'NetworkParameter',
+        key: 'governance.proposal.updateMarket.minEnact',
+        value: '2h0m0s',
+      },
+      {
+        __typename: 'NetworkParameter',
+        key: 'governance.proposal.updateMarket.minProposerBalance',
+        value: '1',
+      },
+      {
+        __typename: 'NetworkParameter',
+        key: 'spam.protection.proposal.min.tokens',
+        value: '1000000000000000000',
+      },
+    ],
+    loading: false,
+    error: undefined,
+  }),
+}));
+
+jest.mock('@apollo/client', () => ({
+  ...jest.requireActual('@apollo/client'),
```

Comment:
chore (blocking): I think in this case it will be simpler (and more thorough) to just provide mock query results to MockedProvider